packages/builder/tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "noEmit": true,
     "jsx": "react-jsx"
   },
-  "include": ["src"]
+  "include": ["src", "../common/src/deterministicJSON.ts"]
```

Comment:
this line will be free for removal once #1340 is merged due to esbuild being enabled.